aframe/src/new-location-based/arjs-device-orientation-controls.js

```diff
@@ -12,6 +12,8 @@
  */
 
 import * as AFRAME from "aframe";
+import * as THREEx from "../../../three.js/build/ar-threex-location-only.js";
```

Comment:
probably is not needed because `ar-threex-location-only.js` is already imported in `gps-new-camera.js` but @nickw1 maybe the code i clearer?

Reply:
Sorry i forgot the issue https://github.com/AR-js-org/AR.js/issues/515 under React, don't consider this comment.

Reply:
@nickw1  If I remove that part, the THREEx reference error still occurs. But the location sample seems to work fine.
So I removed it and committed it.

@kalwalt I upgraded to 1.3.0 as requested and also modified package.json and committed it.

I'm asking because I don't know, do I have to request PR again?